ci-operator/config/openshift/cloud-credential-operator/openshift-cloud-credential-operator-master.yaml

```diff
@@ -100,6 +100,14 @@ tests:
   optional: true
   steps:
     cluster_profile: aws
+    test:
+    - as: e2e-test-sts
+      commands: make test-e2e-sts
+      from: builder
```

Comment:
The error you're seeing is that there is nothing called `builder` in the system - if you meant to reference this base image:

```yaml
  ocp_builder_rhel-8-golang-1.20-openshift-4.14:
    name: builder
    namespace: ocp
    tag: rhel-8-golang-1.20-openshift-4.14
```

I think you need to use the qualified name: `ocp_builder_rhel-8-golang-1.20-openshift-4.14`

Reply:
so complete as in:
```yaml
...
  from: ocp_builder_rhel-8-golang-1.20-openshift-4.14
...
```
and then that would also have the src context as well, that is, both make and the Makefile would be there?